### PR TITLE
Update empower-people-to-work-remotely-remote-access.md

### PR DESCRIPTION
**Pull request by msbemba**
WVD to AVD.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6562

File: microsoft-365/solutions/empower-people-to-work-remotely-remote-access.md [hunk 1]
```diff
@@ -86,16 +86,16 @@ For more information, see this [overview of P2S VPN](/azure/vpn-gateway/point-to
 > [!NOTE]
 > Azure P2S VPN is not included with a Microsoft 365 subscription. You must pay for usage with a separate Azure subscription.
 
-## Deploy Windows Virtual Desktop to provide remote access for remote workers using personal devices
+## Deploy Azure Virtual Desktop to provide remote access for remote workers using personal devices
 
-To support remote workers who can only use their personal and unmanaged devices, use Windows Virtual Desktop in Azure to create and allocate virtual desktops for your users to use from home. Virtualized PCs can act just like PCs connected to your organization network.
+To support remote workers who can only use their personal and unmanaged devices, use Azure Virtual Desktop in Azure to create and allocate virtual desktops for your users to use from home. Virtualized PCs can act just like PCs connected to your organization network.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/empower-people-to-work-remotely-remote-access.md]:
```suggestion
To support remote workers who can only use their personal and unmanaged devices, use Azure Virtual Desktop to create and allocate virtual desktops for your users to use from home. Virtualized PCs can act just like PCs connected to your organization's network.
```

File: microsoft-365/solutions/empower-people-to-work-remotely-remote-access.md [hunk 2]
```diff
@@ -86,16 +86,16 @@ For more information, see this [overview of P2S VPN](/azure/vpn-gateway/point-to
...
 
-![Components of Azure Windows Virtual Desktop.](../media/empower-people-to-work-remotely-remote-access/empower-people-to-work-remotely-remote-access-windows-virtual-desktop.png)
+![Components of Azure Virtual Desktop.](../media/empower-people-to-work-remotely-remote-access/empower-people-to-work-remotely-remote-access-windows-virtual-desktop.png)
 
-For more information, see this [overview of Windows Virtual Desktop](/azure/virtual-desktop/overview).
+For more information, see this [overview of Azure Virtual Desktop](/azure/virtual-desktop/overview).
 
 > [!NOTE]
->Windows Virtual Desktop is not included with a Microsoft 365 subscription. You must pay for usage with a separate Azure subscription.
+>Azure Virtual Desktop is not included with a Microsoft 365 subscription. You must pay for usage with a separate Azure subscription.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/empower-people-to-work-remotely-remote-access.md]:
```suggestion
> Azure Virtual Desktop is not included with a Microsoft 365 subscription. You must pay for usage with a separate Azure subscription.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JoeDavies-MSFT Please merge this PR. Thanks

**Pull request merged**